_rules/audio-or-video-avoids-automatically-playing-audio-80f0bf.md

```diff
@@ -79,6 +76,8 @@ The native `video` and `audio` controls in several browser and assistive technol
 
 The [instruments][instrument] used to pass this rule (if any), must meet all level A Success Criteria in order to fully satisfy [Success Criterion 1.4.2 Audio Control][sc142]. These extra requirements are left out of this rule, and should be tested separately.
 
+User agents do not always automatically play medias, even when an `autoplay` attribute is present. This is done precisely to avoid autoplaying media interrupting the user when they do not want to, especially when the media is likely to contain sound. The decision to respect the `autoplay` attribute or not depends on user settings and previous behavior (interaction with the site, …) Depending on the state of the user agent, failing this rule may cause [Success Criterion 1.4.2 Audio Control][sc142] to be not satisfied.
```

Comment:
This case strikes me as more of an accessibility support issue than an issue needing the requirement to be listed as secondary. The failure test cases below very clearly fail the success criterion, and the only reason that a user would be unaffected would be if the user agent is setup to prevent it.

Reply:
Yes, I agree it should be an AS note, not a Background one, will move it there.
In #2030, both @carlosapaduarte and me experienced that the failure test cases do not autoplay (or not always), and thus were not actually failing the SC (depending on the UA, … and global accessibility support).

In my case, I was building the semi-automated implementation report for Alfa, going to the page for the manual part of the review, seeing that the video did not autoplay and answered so… only to have the report saying that my implementation is incorrect. Only after looking at the code I could see the `autoplay` attribute and the fact that my answer should have been "yes, it autoplays" rather than "no"… But as a user, that video didn't autoplay and the page wasn't failing 1.4.2.

On [CG call of April 13th](https://www.w3.org/2023/04/13-act-r-minutes.html#item09), we discussed it. Especially, around the fact that we should not have failed cases that depend on Accessibility Support (because rules do not consider it directly, so tester A could fail the rule on that case, while tester B passes it, and both are correct because they have different accessibility support). And so we came up with the solution of using it as a secondary requirement only.